docker-compose.yml

```
@@ -0,0 +1,58 @@
+version: '3'
+
+services:
+    homeserver:
+        ### If you already built the Conduit image with 'docker build' or want to use a registry image,
+        ### then you are ready to go.
+        image: c4tplatform/camino-conduit:test
+        ### If you want to build a fresh image from the sources, then comment the image line and uncomment the
+        ### build lines. If you want meaningful labels in your built Conduit image, you should run docker-compose like this:
+        ### CREATED=$(date -u +'%Y-%m-%dT%H:%M:%SZ') VERSION=$(grep -m1 -o '[0-9].[0-9].[0-9]' Cargo.toml) docker-compose up -d
+        build:
+          context: .
```

yamllint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)